src/applications/check-in/components/LanguagePicker.jsx

```diff
@@ -10,8 +10,16 @@ function LanguagePicker(props) {
   const { withTopMargin } = props;
   const { i18n } = useTranslation();
   const { language } = i18n;
+
+  function getUrl(lang) {
+    const url = new URL(window.location.href);
```

Comment:
would it be better to grab this from `router.location`? Feels a little weird to have an absolute url but I don't know that it actually matters...

Reply:
actually it seems like it would take a fair bit of effort since this isn't inside a component with a route.